translation_ab_experiment.py

```
@@ -0,0 +1,499 @@
+        request_payload = request_row.get('request') or {}
+        if dry_run:
+            return VariantRunResult(
+                variant_name=variant_name,
+                settings=settings,
+                dry_run=True,
+            )
+
+        runner = sync_runner or _batch().run_sync_request
+        try:
+            response = runner(request_payload, model_name, api_key_index=api_key_index)
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)